csFastFloat/FastDoubleParser.cs

```diff
@@ -65,6 +65,18 @@ public static unsafe double ParseDouble(string s, chars_format expectedFormat =
       }
     }
 
+    public static unsafe double ParseDouble(string s, out long characters_consumed, chars_format expectedFormat = chars_format.is_general, char decimal_separator = '.')
```

Comment:
```suggestion
    public static unsafe double ParseDouble(string s, out int characters_consumed, chars_format expectedFormat = chars_format.is_general, char decimal_separator = '.')
```
From a .NET point of view `int` is more appropriate, as neither the lenght of a string nor of a ReadOnlySpan can exceed int.MaxValue.